### ISSUE TITLE
[BUG] list-links creates extra links from CamelCase words in caption

### ISSUE THREAD

**Issue by kookma** [Jermolene/TiddlyWiki5#5599]
**Describe the bug**
The list-links macro creates a link to tiddlers from input filter. It shows caption if it is existed instead of title.
The error is occurred when a tiddler caption has missing CamelCase link!

**To Reproduce**
1. Download the attached bundle (list-links-issue.zip) of three tiddlers 
2. Unzip list-links-issue.zip and drag and drop  to  https://tiddlywiki.com/prerelease/
3. Open the Test tiddler
4. See error (the word YouTube is a missing link)

**Expected behavior**
One link expected per title

It seems the in [list-links](https://tiddlywiki.com/prerelease/#%24%3A%2Fcore%2Fmacros%2Flist)

```
<$transclude field="caption">
<$view field="title"/>
</$transclude>
```

shall be replaced with

```
<$view field="caption">
<$view field="title"/>
</$view>
```

I can create a PR if this is correct solution!

**Screenshots**
![image](https://user-images.githubusercontent.com/830394/114272450-77da5280-9a2b-11eb-8257-e61b712a14b9.png)

**TiddlyWiki Configuration (please complete the following information):**
 - Version 5.1.24pre
 - Saving mechanism -
 
**Desktop (please complete the following information):**
 - OS: [e.g. iOS] Windows 10
 - Browser chrome 89

[list-links-issue.zip](https://github.com/Jermolene/TiddlyWiki5/files/6289959/list-links-issue.zip)

**Comment by sobjornstad** [comment]
Just want to add that this happens to me regularly and is really irritating. If the caption is all CamelCase, it is even impossible to go to the intended link target at all.

**Comment by Jermolene** [comment]
I think the best solution here would be for the list-links macro to set the `tv-wikilinks` variable to `no` when rendering the link content, disabling any embedded camelcase links

> It seems the in [list-links](https://tiddlywiki.com/prerelease/#%24%3A%2Fcore%2Fmacros%2Flist)
> 
> ```
> <$transclude field="caption">
> <$view field="title"/>
> </$transclude>
> ```
> 
> shall be replaced with
> 
> ```
> <$view field="caption">
> <$view field="title"/>
> </$view>
> ```

By convention, the caption field is wikified when rendered.

**Comment by flibbles** [comment]
This issue cannot be fixed the way TiddlyWiki is coded right now, so get used to disappointment. This is the world we made for ourselves.

What you can do is add tildas before your TitleCased captions (e.g. "~YouTube: Experience ~TiddlyWiki fluency").

**Comment by kookma** [comment]
> This issue cannot be fixed the way TiddlyWiki is coded right now,

Why not to use https://github.com/Jermolene/TiddlyWiki5/issues/5599#issuecomment-894802530 solution!

**Comment by twMat** [comment]
Is this not the same as issue #1880 ? Therein @Jermolene [indicates](https://github.com/Jermolene/TiddlyWiki5/issues/1880#issuecomment-120930281) that it is desired behaviour.

Also @kookma , if it is the same issue, then your demo tiddlers is really that "YouTube" are separate links which is not a list-links macro as your headline and OP states.

**Comment by Jermolene** [comment]
As far as I can tell, the fix I [suggested](https://github.com/Jermolene/TiddlyWiki5/issues/5599#issuecomment-894802530) works as expected.

I've pushed a fix to the prerelease in 1c16f12d6f5b81d86f79c3e687eec05b3a8d45bf

**Comment by flibbles** [comment]
Oh. You put the link disabling inside the <$link> widget.

I was drunk and grumpy last night, and I missed that obvious solution. My apologies.
